.gitignore

```diff
@@ -64,4 +64,7 @@ src/scripts/pattern-matches/matches_old/
 src/scripts/pattern-matches/generic_matches/
 src/scripts/pattern-matches/upheno_matches/
 src/scripts/pattern-matches/upheno_patterns/
-src/scripts/pattern-matches/ontologies/
\ No newline at end of file
+src/scripts/pattern-matches/ontologies/
+src/ontology/.template.db
```

Comment:
please do not deposit temporary files anywhere outside the `src/ontology/tmp/` directory